recipes/pyraat/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "pyraat" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyraat-{{ version }}.tar.gz
+  sha256: 4d94274dc5fa7b51047b4e7a48e8c3d5a2640714ba6c42b57ee8e686677dd9eb
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - pytest
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - pyraat
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/mmcauliffe/Pyraat
+  summary: Interface for running Praat scripts through Python
+  license: MIT
+  license_file: LICENSE
```

```suggestion
  license_file: 
    - LICENSE
    - LICENSE.txt
```

This includes both license files.